pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -743,7 +782,8 @@ private void setupInput(ContextImpl contextImpl) throws Exception {
         }
     }
 
-    private void setupOutput(ContextImpl contextImpl) throws Exception {
+    @SuppressWarnings({ "unchecked", "rawtypes" })
```

Comment:
Do we need to add these `@SuppressWarnings` as part of this PR?  Seems like a separate issue?

Reply:
No, they are not critical to the functionality of the PR, I just prefer to clean up such errors while I am working on a file since there are rarely, if ever, and tickets related to code cleanup efforts..... So I just do it in a piecemeal fashion as I run across them.